apps/prairielearn/src/lib/config.ts

```diff
@@ -316,6 +316,9 @@ const ConfigSchema = z.object({
       }),
     )
     .default([]),
+  /**
+   * Where uploaded files reside (e.g. uploads from students).
+   */
```

Comment:
It's more nuanced than this; this is only true when specifically using the `FileSystem` file store, and it's not limited to uploaded files (see e.g. `apps/prairielearn/src/pages/instructorFileTransfer/instructorFileTransfer.ts`).

Not to sound like a broken record, but every change to this PR is opening up new avenues of discussion and making it harder to merge the _actual_ changes here. I really appreciate the effort to improve things. I don't want to discourage that, but we've now spiraled from one changed file into 19. Let's focus on merging the improvements to the developer documentation and integrate other improvements later.

Reply:
Reverted in d8a9ff4198737ef4fefa15f55a610afb378d9154 and split into https://github.com/PrairieLearn/PrairieLearn/issues/11970